_data/mentors.yml

```
@@ -1018,4 +1018,29 @@
     mentee: A mentee who wants learn about backend architecture. Learn how to design and implement RESTful APIs.
     extra: Understanding business value of the code, RESTful API design, Breaking down a problem, Importance of clean code and documentation, Career change, Returning to work after a break, Agile workflow and sprints (developer's prespective)
   network:
-    - linkedin: https://www.linkedin.com/in/madhurachaganty
\ No newline at end of file
+    - linkedin: https://www.linkedin.com/in/madhurachaganty
+
+- name: Mira Davidson
+  hours: 2
+  position: Software Engineer, Labstep
+  type: both
+  index: 37
+  location: United Kingdom, London
+  bio: I have 2.5 years of experience working at SaaS startups in the biotech sector. My background is in biochemistry and I transitioned into software engineering by combining the two. Now I work on mostly frontend development in React at Labstep. I have also worked as a researcher in ML/AI and built software tools in an academic environment.
+  image: assets/images/mentors/4.jpeg
```

the name of the photo should be the index of the mentor. Please adjust that. 

```suggestion
  image: assets/images/mentors/37.jpeg
```